osu.Game/Graphics/OsuColour.cs

```diff
@@ -266,7 +266,7 @@ public static Color4 ForegroundTextColourFor(Color4 backgroundColour)
         public readonly Color4 Yellow = Color4Extensions.FromHex(@"ffcc22");
         public readonly Color4 YellowDark = Color4Extensions.FromHex(@"eeaa00");
         public readonly Color4 YellowDarker = Color4Extensions.FromHex(@"cc6600");
-
+        public readonly Color4 GreenLightest = Color4Extensions.FromHex(@"00ffaa");
```

Comment:
Where did this come from?

Reply:
I have moved the color that the tab buttons were using into that file